feature-governance-impl/src/main/java/io/novafoundation/nova/feature_governance_impl/presentation/tindergov/cards/TinderGovCardsViewModel.kt

```diff
@@ -305,9 +355,27 @@ class TinderGovCardsViewModel(
 
             if (isBasketItemRemoved) {
                 _resetCards.sendEvent()
+                loadFirstCards()
             }
         }
     }
+
+    private fun createCardsBackground(): List<Int> {
+        return listOf(
+            R.drawable.tinder_gov_card_background_1,
+            R.drawable.tinder_gov_card_background_2,
+            R.drawable.tinder_gov_card_background_3,
+            R.drawable.tinder_gov_card_background_4,
+            R.drawable.tinder_gov_card_background_5,
+            R.drawable.tinder_gov_card_background_6,
+            R.drawable.tinder_gov_card_background_5,
+            R.drawable.tinder_gov_card_background_4,
+            R.drawable.tinder_gov_card_background_3,
+            R.drawable.tinder_gov_card_background_2,
+            R.drawable.tinder_gov_card_background_1,
+            R.drawable.tinder_gov_card_background_0,
+        )
+    }
```

Comment:
Why do we use 12345654321 insteaed of just 123456?

Reply:
To have the color loop according to design: 
![image](https://github.com/user-attachments/assets/d322cfc2-c1a7-494c-ac8b-2aa584fa7353)